### PR TITLE
Make kpi buttons size dependant on font size

### PR DESCRIPTION
**Pull request by daren-thomas**
@christophwaibel this addresses #416 (visualizer form autoscale KPI buttons). Would you mind testing it on your computer to see if it looks ok there too?

### CODE REVIEW

File: src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs [hunk 1]
```diff
@@ -12,15 +12,12 @@ namespace Hive.IO.GhInputOutput
...
     public class GhVisualizerAttributes : GH_ResizableAttributes<GhVisualizer>
     {
-        // make sure we have a minimum size
-        public static float TitleBarHeight = 200f;
-        private static readonly float MinWidth = 200f;
-        private static readonly float MinHeight = TitleBarHeight + 150f;
-
         private const int Padding = 6;
+        private static readonly float MinWidth = 1144f;
+        private static readonly float MinHeight = TitleBarHeight + 580;
```
Comment (christophwaibel) [inline src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs:17]: can you half, or even third, these min width and height values, please? the component is too big in its smallest form

**Review by christophwaibel — CHANGES_REQUESTED** [review]
looks alright, much better sizes of these buttons. 
two issues: 
- min component size too big
- the black bar indicator for the selected KPI button  covers half of the text:
image

how about you change the design to a black frame around the selected button instead?

image

File: src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs (hunk 1 above)
Comment (daren-thomas) [reply src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs]: OK! I've pushed some changes, please check it out on your screen for me and post the results:

minimum size:
image

selection bar for kpi:
image

Comment (christophwaibel) [reply src/Hive.IO/Hive.IO/GhInputOutput/GhVisualizerAttributes.cs]: thanks, looks about the same now

**Review by christophwaibel — APPROVED** [review]

**Pull request merged**